app/libs/presenters/user.rb

```diff
@@ -59,6 +59,65 @@ def dashboard_name_link
       url_helpers.root_path
     end
 
+    def show_when_subscribe?(as: , field:)
+      return true if field.in?(subscribe_fields(as: as))
```

Comment:
juste : field.in?(subscribe_fields(as: as))

Reply:
exact !